### ISSUE TITLE
Link shows target = "blank" in html text

### ISSUE THREAD

**Issue by JanaJarecki** [dsifford/academic-bloggers-toolkit#549]
**ABT Version:**  4.13.3
**PHP Version:** PHP 7.0
**Theme:** Photoreactive
**Plugin List:**
[note: after disabling all plugins, the issue remained]
- AGW: COP PDF Attachment Menu
- Category Posts Widget
- Contact Form 7
- Custom Twitter Feeds
- Easy Footnotes
- Google Analytics Dashboard for WP (GADWP)
- iMaginem Fullscreen Post Creater (comes with the theme)
- iMaginem Portfolio Creater
- iMaginem Shortcodes Generator
- iMaginem Widget Set
- MathJax-LaTeX
- Mendeley Plugin
- WordPress MU Domain Mapping
- WP Code Highlight.js

**Browser:**  Google Chrome | 72.0.3626.121
**Expected behavior:**  the html tags to be included in the <a href ...> issue
**Actual behavior:** see here: http://janajarecki.com/research/
The reference shows the html tags in the text, as shown here
![image](https://user-images.githubusercontent.com/9809311/53951134-1027df80-40ce-11e9-8b58-ca6b18607352.png)

**Console messages:** no errors in console, this is the html
![image](https://user-images.githubusercontent.com/9809311/53951217-42d1d800-40ce-11e9-9654-1db9993ff2e3.png)

**Comment by dsifford** [comment]
Sorry for the delay in getting back with you @JanaJarecki.

This is indeed strange. Can you share with me the following info...

1. What is the citation style you are using?
2. How did you enter the reference in question that caused this problem?

Thanks! Sorry for the trouble!

**Comment by glorious1** [comment]
Looks like she solved the problem, as the link shows correctly now in the page she links to.

**Comment by JanaJarecki** [comment]
@glorius1: The bug remains. Why you don't see it is because the site now
uses "APA with doi", which you can see from the doi prefix in front of the
link. See my last comment on the style.

Let me know if you have a fix and I'll switch the style back to test. I can
switch back to the bug temporarily in coordination if it helps debugging.

glorious1 <notifications@github.com> schrieb am Sa., 16. März 2019, 19:26:

**Comment by dsifford** [comment]
@JanaJarecki If you could, please address the questions I asked and I'll look into this further.

**Comment by JanaJarecki** [comment]
Hi,
I did answer you via the email answering, the message seems to have gotten lost, I'll copy-paste my answer from 14. March:

HI!
thank you for the reply and taking the time. Sure:

I entered the reference via doi.
The style is "American Psychological Association 6th edition".
Note, the error disappears when the style is switched to "American Psychological Association 6th edition *("doi:" DOI prefix)*".


Best
Jana
